tests/handlers/filesystem/test_jffs2.py

```diff
@@ -57,6 +59,10 @@ def get_valid_jffs2_old_be_header():
     )
 
 
+def calculate_crc(header: Instance):
+    return (binascii.crc32(header.dumps()[:-4], -1) ^ -1) & 0xFFFFFFFF
```

Comment:
Why we are dropping the last 4 bytes? Naming that number would be more readable.